blog/content/post/2021-01-27-fosdem2021.md

```diff
@@ -0,0 +1,125 @@
+FOSDEM 2021 as a continuation of this devroom of the previous edition. We take
+part in this event to promote the use of free and open source
+software, and show how it can increase security and the quality of the code
+itself and give more business freedom outside of the vendors lock-in, from the
+point of the firmware level which is our daily bread in 3mdeb. In this blogpost
+we will gather all the information about the event and links to the videos from
+the fosdem 2020.
+
+> FOSDEM is a free event for software developers to meet, share ideas and
+collaborate. Every year, thousands of developers of free and open source
+software from all over the world gather at the event in Brussels. In 2021, they
+will gather online.
+
+### Why Open source
+
+Before diving into the technical information around fosdem, let's make a step
+back and refresh the ground knowledge that made us choose the Open Source path
+as users, contributors, developers, supporters – OSS community members.
+
+* Open source builds trust in the features and quality of the software. If the
+company wants to start developing OS project, it needs to check its quality. The
+published code's quality can be easily scanned by having full insight into the
+source code.
+* Developers are building a visible portfolio by contributing to the Open Source
+project. The company that decided to develop a particular project is more eager
+to hire contributors with proven expertise.
+* The company can get improved performance for a relatively short amount of time.
+ Having open-source code base allows to prove the competencies and gain trust
+ without any additional audits.
+* The source code has no value itself. Most of the products are made by modifying
+existing code components, so technology knowledge that allows performing on the
+ready-to-use code in the proposed business model brings the essential value.
+
+### Where to link the event?
+
+* To participate, no registration is needed. Check the [schedule](https://fosdem.org/2021/schedule/track/open_source_firmware_bmc_and_bootloader/) and choose the talks you would take part in. We are in [D.firmware](https://fosdem.org/2021/schedule/room/dfirmware/) room.
+* Find the live streaming for a D.firmware [here](https://fosdem.org/2021/schedule/streaming/). The page will be updated every few minutes to show you what is currently
+scheduled in each room.
+
+### How we are proceeding this year
+
+This online event will take place on 6 & 7 February 2021. Here you will find [the full schedule](https://fosdem.org/2021/schedule/track/open_source_firmware_bmc_and_bootloader/).
+The below points will be updated with recordings after the event.
+
+##### February 6 2020
+
+* [Open Source Firmware status on AMD platforms 2021](https://fosdem.org/2021/schedule/event/firmware_osfsoap2/) by [Piotr Król](https://twitter.com/pietrushnic) and [Marek Kasiewicz](https://fosdem.org/2021/schedule/speaker/marek_kasiewicz/)
```

Comment:
This schedule contains moderators: Marek, Jakub, Norbert and Cezary. They shouldn't be mentioned along with speakers. At least mark them as moderators or remove them. This is caused by the way Pentabarf interpret other roles, which is weird and incorrrect

Reply:
ok, thank you for mentioning, I will remove moderators